examples/story-example.cuentitos

```diff
@@ -2,88 +2,88 @@ You've just arrived in the bustling city, full of excitement and anticipation fo
 The skyline reaches for the clouds, and the sounds of traffic and people surround you. 
 As you take your first steps in this urban jungle, you feel a mix of emotions, hoping to find your place in this new environment.
 	* I take a walk through a nearby park to relax and acclimate to the city.
-    As you stroll through the nearby park, the soothing sounds of rustling leaves and chirping birds help calm your senses. 
-    You find a quiet bench to sit on, taking a moment to breathe deeply and gather your thoughts. 
-    (50%) A solitary figure sits on the bench, engrossed in a book, seemingly unfazed by the surrounding city noise.
-    This serene oasis gives you the confidence to face the challenges ahead as you acclimate to your new life in the bustling city.
+		As you stroll through the nearby park, the soothing sounds of rustling leaves and chirping birds help calm your senses. 
```

Comment:
Wait. Is the problem with spaces and tabs at the same time, or with using two spaces? 
I rather use spaces instead of tabs.